verification/curator-service/ui/src/components/App.tsx

```diff
@@ -310,10 +317,11 @@ export default function App(): JSX.Element {
     const location = useLocation<LocationState>();
     const menuList = [
         {
-            text: 'Home',
+            text: 'Charts',
             icon: <HomeIcon />,
             to: '/',
-            displayCheck: (): boolean => true,
+            // Readers only see the linelist, home button is useless to them.
+            displayCheck: (): boolean => user !== undefined && !isJustReader(),
```

Comment:
Seems simpler as `hasAnyRole(['curator', 'admin'])` and matches the other displayCheck logic